README.md

```diff
@@ -71,8 +71,9 @@ The content type of the response depends on the request's `Accepts` header.
 
 -  For Accepts header `json` or `application/json`, the response content type is JSON.
 -  For Accepts header `html` or `text/html`, the response content type is HTML.
+-  For Accepts header `xml` or `xml`, the response content type is XML.
```

Comment:
`xml` is repeated twice, should the second example say `text/xml` or `application/xml` (or preferably both) instead?